src/components/pages/HomePage/HomePage.module.scss

```
@@ -175,24 +170,53 @@
 
       .text-content {
         width: 70%;
-      }
 
+        @include media(">=#{$desktop-hr}") {
+          width: 64%;
+        }
+      }
     }
 
-    >.methodology-list {
+    > .methodology-list {
       margin-top: 8px;
       padding-left: $content-horizontal-padding;
 
       @include desktop {
         padding: 52px 0;
 
         grid-column-gap: 4vw;
+
+        @include media(">=#{$desktop-hr}") {
+          grid-column-gap: 88px;
```

`grid-column-gap` is not standard.

```suggestion
          column-gap: 88px;
```

---

Actually, we've been using `grid-column-gap` (7) more often than `column-gap` (1). Do you want to switch all occurrences to `column-gap`?

---

Done!